app/src/main/java/app/revanced/manager/ui/components/Dialogs.kt

```diff
@@ -0,0 +1,206 @@
+                TextButton(onClick = { onPopupDismissed() }) {
+                    Text(text = "Close")
+                }
+            },
+            title = {
+                Text(
+                    text = stringResource(R.string.app_version)
+                )
+            },
+        )}
+
+}
+
+@Composable
+fun HelpDialog() {
+
+    var showPopup by remember { mutableStateOf(false) }
+
+    val onPopupDismissed = { showPopup = false }
+
+    var currentUriHandler = LocalUriHandler.current
+
+    PreferenceRow(
+        title = stringResource(R.string.help),
+        painter = painterResource(id = R.drawable.ic_baseline_help_24),
+        onClick = { showPopup = true },
+    )
+
+    if (showPopup) {
+        AlertDialog(
+            backgroundColor = MaterialTheme.colorScheme.background,
+            onDismissRequest = onPopupDismissed,
+            text = {
+                Column(Modifier.padding(8.dp)) {
+                    Text(text = "In need of some help?\nJoin our Discord Server and ask in our dedicated support channel!")
+                }
+            },
+            confirmButton = {
+                TextButton(onClick = { currentUriHandler.openUri("https://discord.gg/mxsFc6nyqp") }) {
+                    Text(text = "Open Discord")
+                }
+            },
+            dismissButton = {
+                TextButton(onClick = { onPopupDismissed() }) {
+                    Text(text = "Close")
+                }
+            },
+            title = {
+                Text(
+                    text = stringResource(R.string.help)
+                )
+            },
+        )}
+
+}
+
+
+@Composable
+fun FAQDialog(
+) {
+    var showPopup by remember { mutableStateOf(false) }
+
+    PreferenceRow(
+        title = stringResource(id = R.string.faq),
+        onClick = { showPopup = true },
+        painter = painterResource(id = R.drawable.ic_faq),
+    )
+
+    if(showPopup) {
+        Box(
+            Modifier.padding(vertical = 32.dp)
+        ) {
+            Dialog(
+                properties = DialogProperties(
+                    dismissOnBackPress = true,
+                    dismissOnClickOutside = true,
+                ),
+                onDismissRequest = {
+                    showPopup = false
+                }
+            ) {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    shape = RoundedCornerShape(16.dp),
+                    color = MaterialTheme.colorScheme.background
+                ) {
+                    Box(
+                        contentAlignment = Alignment.Center,
+                        modifier = Modifier
+                            .padding(horizontal = 8.dp, vertical = 32.dp)
+                            .verticalScroll(
+                                rememberScrollState()
+                            )
+                    ) {
+                        Text(
+                            text =
```

Comment:
This will also be required to fetch from the api.

Reply:
> > This will also be required to fetch from the api.
> 
> the invite here is different from the invite on the website (revanced.app/discord), this one redirects you directly to the #support channel

If the user doesn't get verified, they also can't see the support channel.